pkgs/development/tools/build-managers/buildbot9/plugins.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv,
+  fetchurl,
+  buildPythonApplication,
+  python
+}:
+
+{
+  www = buildPythonApplication rec {
+    name = "buildbot-www-0.9.0rc1";
+    namePrefix = "";
+    format = "wheel";
```

Comment:
Building from source requires the buildbot package to be installed. This gives us a circular dependency and the result folder has no bin/ in it after building.

Do you have any ideas on how to handle this outside of using the wheel?